### PR TITLE
allure-python 2.13.2 (ursaverance effort)

### PR DESCRIPTION
**Pull request by lorepirri**
[PKG-2589] allure-python 2.13.2 (ursaverance effort)

- linter
- remove `noarch`
- `about` section

**Depends on**
- AnacondaRecipes/behave-feedstock#2

[PKG-2589]: https://anaconda.atlassian.net/browse/PKG-2589?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
The package seems to be failing in almost all architectures.

**Review by cbouss — APPROVED** [review]
LGTM
You could have setuptools-scm unpinned.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,115 +11,126 @@ source:
...
-      noarch: python
     requirements:
       host:
         - pip
-        - python >=3.6
-        - setuptools-scm
+        - python
+        - setuptools
+        - wheel
+        - setuptools-scm 7.1.0
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
        - setuptools-scm
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,115 +11,126 @@ source:
...
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
        - setuptools-scm
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,115 +11,126 @@ source:
...
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
        - setuptools-scm
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -11,115 +11,126 @@ source:
...
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
        - setuptools-scm
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -11,115 +11,126 @@ source:
...
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
        - setuptools-scm
```

File: recipe/meta.yaml [hunk 6]
```diff
@@ -11,115 +11,126 @@ source:
...
-      noarch: python
     requirements:
       host:
         - pip
-        - python >=3.6
-        - setuptools-scm
+        - python 
+        - setuptools
+        - wheel
+        - setuptools-scm 7.1.0
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
        - setuptools-scm
```

**Comment by lorepirri** [comment]
> LGTM You could have setuptools-scm unpinned.

thank you @cbouss , I've unpinned all of the `setuptools-scm`

File: recipe/meta.yaml [hunk 7]
```diff
@@ -11,115 +11,126 @@ source:
...
+        - python
         - attrs >=16.0.0
         - pluggy >=0.4.0
-    script: install.sh
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
     test:
       imports:
         - allure
         - allure_commons
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:35]:
```suggestion
        - allure_commons
        
```
For readability. And the same for other outputs

File: recipe/meta.yaml [hunk 8]
```diff
@@ -11,115 +11,126 @@ source:
...
+        - wheel
         - setuptools-scm
       run:
-        - python >=3.6
+        - python
         - attrs >=16.0.0
         - pluggy >=0.4.0
-    script: install.sh
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Could you place these lines under   `- name: allure-python-commons` and correspondingly for other outputs?

File: recipe/meta.yaml [hunk 9]
```diff
@@ -11,115 +11,126 @@ source:
...
   home: https://github.com/allure-framework/allure-python
+  dev_url: https://github.com/allure-framework/allure-python
   license: Apache-2.0
   license_file: LICENSE
+  license_family: APACHE
   summary: Allure integrations for Python test frameworks
-
+  description: |
+    Allure Framework is a flexible lightweight multi-language test report tool that not only shows a very concise representation of what have been tested in a neat web report form, but allows everyone participating in the development process to extract maximum of useful information from everyday execution of tests.
+  doc_url: https://allurereport.org
```
Comment (Jrice1317) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://allurereport.org/docs/
  
```

File: recipe/meta.yaml [hunk 10]
```diff
@@ -11,115 +11,126 @@ source:
   sha256: 9db4db2133a52e48af043fbb3bb920a77886f64a6a1c8cd56adee41828487fc6
 
 build:
-  number: 1
+  number: 0
+  skip: true  # [py<38]
 
 outputs:
   - name: allure-python-commons
```
Comment (Jrice1317) [inline recipe/meta.yaml:18]:
```suggestion
  - name: allure-python-commons
    script:
       - install.sh   # [unix]
       - install.bat  # [win]
```

Comment (Jrice1317) [reply recipe/meta.yaml:18]: Something like this? @skupr-anaconda

Comment (skupr-anaconda) [reply recipe/meta.yaml:18]: Yep

Comment (lorepirri) [reply recipe/meta.yaml:18]: thank you @Jrice1317 and @skupr-anaconda

File: recipe/meta.yaml (hunk 8 above)
Comment (lorepirri) [reply recipe/meta.yaml]: done

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**